sunpy/map/mapbase.py

```diff
@@ -672,10 +680,12 @@ def unit(self):
 
     def _base_name(self):
         """Abstract the shared bit between name and latex_name"""
-        return "{nickname} {{measurement}} {date}".format(
-            nickname=self.nickname,
-            date=parse_time(self.date).strftime(TIME_FORMAT)
-        )
+        if self.measurement is None:
+            format_str = "{nickname} {date}"
+        else:
+            format_str = "{nickname} {{measurement}} {date}"
+        return format_str.format(nickname=self.nickname,
+                                 date=parse_time(self.date).strftime(TIME_FORMAT))
```

Comment:
This is shorter and slightly less repetitive, but is it actually better??

```suggestion
        measurement_str = "{{measurement}} " if measurement else ""
        format_str = "{nickname} {measurement_str}{date}"
        return format_str.format(nickname=self.nickname,
                                 measurement=measurement_str
                                 date=parse_time(self.date).strftime(TIME_FORMAT))
```

Reply:
Not want a space between `{measurement_str}{date}`?

Reply:
No, the space is conditionally in `measurement_str`.

Reply:
I think I prefer my version, but not too fussed.